model_prices_and_context_window.json

```diff
@@ -39,61 +54,81 @@
         "input_cost_per_token": 0.000003,
         "output_cost_per_token": 0.000004
     },
-    "gpt-3.5-turbo-instruct": {
-        "max_tokens": 8192,
-        "input_cost_per_token": 0.0000015,
+    "text-ada-001": {
```

Comment:
@krrishdholakia I have added the models to https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json. Check it out under "Files changed"  -> model_prices_and_context_window.json

Reply:
why is gpt-3.5-turbo-instruct removed? @WilliamEspegren 

can you revert any changes to `__init__.py`

Reply:
The gpt-3.5-turbo-intruct model is included check the file. I can revert changes made to __ini__.py (mostly just autopep8 formatting)

Reply:
@krrishdholakia The changes to __init__.py are reverted